docs/sources/operators-guide/tools/mimirtool.md

```diff
@@ -199,13 +199,13 @@ To perform a trial run that does not make changes, you can use the dry run flag
 mimirtool rules lint <file_path>...
 ```
 
-The format of the file is the same format as shown in [rules load](#load).
+The format of the file is the same format as shown in [rules load](#load-rule-groups).
 
 #### Prepare
 
 This `prepare` command prepares a rules file that you upload to Grafana Mimir.
 It lints all PromQL expressions and adds a label to your PromQL query aggregations in the file.
-The format of the file is the same format as shown in [rules load](#load).
+The format of the file is the same format as shown in [rules load](#load-rule-groups).
```

Comment:
I don't see where `#load-rule-groups` should point in this document. Also sometimes we use `{{< relref "#..." >}}`, which one is correct?

Reply:
There's no practical difference when it comes to self-page anchors.

Reply:
I thought `relref` links get checked, but if there is no difference, I'd suggest to use `[...](#...)` form everywhere, as it doesn't confuse other tools (eg. Goland). But that's not for this PR.

Reply:
(Internal) See https://github.com/grafana/technical-documentation/issues/404.

Reply:
@gguillotte-grafana, there might or might not be link checking for `ref`, `relref`, or both.

Reply:
Both ways to define an anchor are supported by [doc-validator](https://github.com/grafana/technical-documentation/tree/main/tools/doc-validator) (I've written the original one, then migrated to `grafana/technical-documentation` repo).